internal/controller/operator/eventing/integrationtests/natsconnection/integration_test.go

```diff
@@ -21,6 +21,8 @@ import (
 )
 
 // Test_NATSConnection tests the Eventing CR status when connecting to NATS.
+//
+//nolint:tparallel // lets not make this test parallel for now
```

Comment:
I still see `t.Parallel()` in the test line `#91`. Was it failing the `tparallel` linter?